patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -53,43 +53,43 @@
 		"ModsShowAllMods": "Tout montrer",
 		"ModsShowEnabledMods": "Montrer uniquement les Mods actifs",
 		"ModsShowDisabledMods": "Montrer uniquement les Mods inactifs",
-		// "ModsXModsFilteredByEnabled": "{0} {^0:mod;mods} filtered by enabled filter.",
-		// "ModsXModsFilteredByModSide": "{0} {^0:mod;mods} filtered by enabled mod side filter.",
-		// "ModsXModsFilteredBySearch": "{0} {^0:mod;mods} filtered by enabled search filter.",
+		// "ModsXModsFilteredByEnabled": "{0} {^0:mod;mods} filtré par filtre activé.",
+		// "ModsXModsFilteredByModSide": "{0} {^0:mod;mods} filtré par filtre côté mod activé.",
+		// "ModsXModsFilteredBySearch": "{0} {^0:mod;mods} filtré par filtre de recherche activé.",
 		"ModsXItems": "{0} {^0:objet;objets}",
 		"ModsXNPCs": "{0} {^0:PNJ;PNJs}",
 		"ModsXTiles": "{0} {^0:block;blocks}",
 		"ModsXWalls": "{0} {^0:mur;murs}",
 		"ModsXBuffs": "{0} {^0:buff;buffs}",
 		"ModsXMounts": "{0} {^0:monture;montures}",
-		// "ModsReloadAndReturn": "Reload and return to world menu",
-		// "ModsNotFoundServer": "No mods were found in: \"{0}\"\nIf you are running a dedicated server, you may wish to use the 'modpath' command line switch or server config setting to specify a custom mods directory.\n",
-		// "AskForModIndex": "Type a number to switch between enabled/disabled",
-		// "AskForCommand": "Type a command: ",
+		// "ModsReloadAndReturn": "Recharger et retourner au menu du monde",
+		// "ModsNotFoundServer": "No mods were found in: \"{0}\"\nSi vous exécutez un serveur dédié, vous pouvez utiliser le commutateur de ligne de commande 'modpath' ou le paramètre de configuration du serveur pour spécifier un répertoire de mods personnalisés.\n",
```

Comment:
Shouldn't  "No mods were found in: \" translated by "Aucun Mod n'a été trouvé à :\" ?